Doc/library/types.rst

```diff
@@ -132,7 +132,7 @@ Standard names are defined for the following types:
    .. versionadded:: 3.6
 
 
-.. data:: CodeType
+.. class:: CodeType(**kwargs)
```

Comment:
I would suggest to leave the constructor undocumented as it was before, so new people don't rely con constructing code objects by hand.

Reply:
I needed to change this so the docs rendered properly as a method -- do you have a suggestion on how to do that without changing this?

the fact of the matter is people are already constructing `CodeType` objects by hand (which was part of the motivation for the `.replace(...)` method as far as I can tell to make that _easier_)

looking forward to your suggestion :)

Reply:
> I needed to change this so the docs rendered properly as a method -- do you have a suggestion on how to do that without changing this?

In that case don't worry, I don't think is worth the trouble, specially if we use `**kwargs` :)

>  to make that easier

The idea is that they don't need to use the constructor so they don't break because we introduce new parameters if they just want to change 1 or 2 of them, which is the common scenario.